### ISSUE TITLE
Automated tests fail due to ha.pool.sks-keyservers.net unavailability

### ISSUE THREAD

**Issue by m-strzelczyk** [googleapis/python-compute#79]
#### Environment details

  - OS type and version: n/a
  - Python version: `python --version`: n/a
  - pip version: `pip --version`: n/a
  - `google-cloud-compute` version: `pip show google-cloud-compute`: n/a

#### Steps to reproduce

  1. Try to make a Pull Request, so kokoro tries to build like, just like: https://github.com/googleapis/python-compute/pull/78
  2. The docs-presubmit test fails during its setup.

#### Code example

None needed.

#### Stack trace

From the test output:
```
Step 6/8 : RUN set -ex     && export GNUPGHOME="$(mktemp -d)"     && echo "disable-ipv6" >> "${GNUPGHOME}/dirmngr.conf"     && /tmp/fetch_gpg_keys.sh     && for PYTHON_VERSION in 3.7.8 3.8.5; do         wget --no-check-certificate -O python-${PYTHON_VERSION}.tar.xz "https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz"         && wget --no-check-certificate -O python-${PYTHON_VERSION}.tar.xz.asc "https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz.asc"         && gpg --batch --verify python-${PYTHON_VERSION}.tar.xz.asc python-${PYTHON_VERSION}.tar.xz         && rm -r python-${PYTHON_VERSION}.tar.xz.asc         && mkdir -p /usr/src/python-${PYTHON_VERSION}         && tar -xJC /usr/src/python-${PYTHON_VERSION} --strip-components=1 -f python-${PYTHON_VERSION}.tar.xz         && rm python-${PYTHON_VERSION}.tar.xz         && cd /usr/src/python-${PYTHON_VERSION}         && ./configure             --enable-shared             --enable-unicode=ucs4             --with-system-ffi             --without-ensurepip         && make -j$(nproc)         && make install         && ldconfig   ; done   && rm -rf "${GNUPGHOME}"   && rm -rf /usr/src/python*   && rm -rf ~/.cache/
 ---> Running in 39d592cf26a8
+ mktemp -d
+ export GNUPGHOME=/tmp/tmp.vjBaqEwlHa
+ echo disable-ipv6
+ /tmp/fetch_gpg_keys.sh
gpg: keybox '/tmp/tmp.vjBaqEwlHa/pubring.kbx' created
gpg: keyserver receive failed: No name
command failed, retrying
gpg: keyserver receive failed: No name
command failed, retrying
gpg: keyserver receive failed: No name
command failed, retrying
gpg: keyserver receive failed: No name
command failed, retrying
gpg: keyserver receive failed: No name
command failed, retrying
gpg: keyserver receive failed: No name
command failed, retrying
The command '/bin/sh -c set -ex     && export GNUPGHOME="$(mktemp -d)"     && echo "disable-ipv6" >> "${GNUPGHOME}/dirmngr.conf"     && /tmp/fetch_gpg_keys.sh     && for PYTHON_VERSION in 3.7.8 3.8.5; do         wget --no-check-certificate -O python-${PYTHON_VERSION}.tar.xz "https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz"         && wget --no-check-certificate -O python-${PYTHON_VERSION}.tar.xz.asc "https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz.asc"         && gpg --batch --verify python-${PYTHON_VERSION}.tar.xz.asc python-${PYTHON_VERSION}.tar.xz         && rm -r python-${PYTHON_VERSION}.tar.xz.asc         && mkdir -p /usr/src/python-${PYTHON_VERSION}         && tar -xJC /usr/src/python-${PYTHON_VERSION} --strip-components=1 -f python-${PYTHON_VERSION}.tar.xz         && rm python-${PYTHON_VERSION}.tar.xz         && cd /usr/src/python-${PYTHON_VERSION}         && ./configure             --enable-shared             --enable-unicode=ucs4             --with-system-ffi             --without-ensurepip         && make -j$(nproc)         && make install         && ldconfig   ; done   && rm -rf "${GNUPGHOME}"   && rm -rf /usr/src/python*   && rm -rf ~/.cache/' returned a non-zero code: 1
```

Basically, those two commands are failing:
```
gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D
gpg --keyserver ha.pool.sks-keyservers.net --recv-keys E3FF2839C048B25C084DEBE9B26995E310250568
```

I checked that `ha.pool.sks-keyservers.net` fails to resolve. Do we really want to validate python build like this? Isn't there a better, more reliable way?

As we can see at: https://sks-keyservers.net/overview-of-pools.php
```
This service is deprecated. This means it is no longer maintained, and new HKPS certificates will not be issued. Service reliability should not be expected.

Update 2021-06-21: Due to even more GDPR takedown requests, the DNS records for the pool will no longer be provided at all.
```

**Comment by vchudnov-g** [comment]
Is this fixed? @m-strzelczyk @parthea

**Comment by busunkim96** [comment]
Yes, this was fixed in https://github.com/googleapis/python-compute/pull/67. Closing.
